client/src/App.tsx

```diff
@@ -162,19 +163,29 @@ function App() {
     <>
       <Header />
       <PlayerScore
-        playerName={player1.name}
-        playerId={player1.id}
-        updateName={() => {}}
-        playerScore={player1.score}
+        name={
+          inputPlayerNames.find(({ id }) => id === player1.id)?.name ??
+          player1.name
+        }
+        id={player1.id}
+        updateName={updatePlayerName}
+        validateName={validatePlayerName}
+        score={player1.score}
         currentRound={currentRoundRef.current}
         totalRounds={totalRounds}
         roundWinners={roundWinners}
       />
+      <script></script>
+      Player 1 name in game object is: {player1.name}
```

Comment:
This line can be removed?

Reply:
Ah yes, the Player 1 name etc was just for demonstrating that the name was stored differently in the game state. The script I think was an accidental copy paste. Will remove and push again now.

Reply:
Done